### DISCUSSION TITLE
identify if column is documented

### DISCUSSION THREAD

**Discussion by jaredchurch** [dbt-labs/dbt-core#10669] [Ideas]
We're in the process of driving improvement in the level of documentation in our dbt project and there are a couple of things that I'm looking for a way to do...

1. We use snowflake as our database and I wonder if there is a way to include comments in snowflake (table/column) based on the documentation in dbt yml file? I've seen some places where this is done with tools, but know very little about it.
2. I'm wondering if there is a way that I can measure which columns do an don't have documentation in yml file - effectively working as a QA metric for the team to identify gaps in documentation. I see this almost fitting into category of linting operation.

Doing a little google research I cannot see anything obvious and wanted to ask before I go looking at figuring out some kind of bespoke solution for these.

**Comment by jaredchurch** [comment]
I just got pointed here as a possible solution for this...
https://docs.getdbt.com/reference/resource-configs/persist_docs

**Reply by jaredchurch** [reply]
ok, you can ignore me - this documentation gives me a capability that works perfectly for my needs.

**Comment by pgoslatara** [comment]
Regarding 2, there are a few options you can look into:
* [dbt-coverage](https://github.com/slidoapp/dbt-coverage): report on your documentation coverage.
* [dbt-bouncer](https://github.com/godatadriven/dbt-bouncer): enforce your documentation coverage.

Disclaimer: I'm a contributor to `dbt-coverage` and the author of `dbt-bouncer`.
